docs/dm/assets.md

```diff
@@ -87,62 +98,72 @@ This menu includes a plethora of options.
 
 ### Is a token
 
-Selecting this option classifies the asset as a token. 
-This will help you keep track of which assets are tokens and which are other assets.
+Selecting this option classifies the object as a token. 
+Tokens function fundamentally differently than other shapes.
+
+Tokens cannot pass through shapes with the 'blocks movement' option enabled. 
+Tokens are also effected by lighting.
+Check the [lighting and vision](/docs/dm/light-shadows/) document for more information on how lighting works in relation to tokens.
 
 ### Is invisible
 
-Selecting this option turns the asset invisible to your players.
-As the DM, you will still be able to see the asset.
+Selecting this option turns the shape invisible to your players.
```

Comment:
```suggestion
Selecting this option turns the shape invisible to all players, except those with vision or edit access to the shape.
```
This is more or less an "equal" feature in a sense that it can and should both be used by players and DMs.